azure/README.md

```diff
@@ -80,9 +85,13 @@ or **Contributor** role on the Subscription to create a new Resource Group for t
 Many communities keep their data lake files on Azure Files. This is optional.
 
 1. **To use Azure Files:**
-   - Create the Storage Account and Files share beforehand
+   - Create the Storage Account and Files share beforehand, or do so while deploying the VM
    - The `cloud-config.yml` already contains the mount script with ARM parameter placeholders
    - You'll provide the storage account details as parameters during deployment
+   - CMI uses the following naming conventions:
+     - Storage Account Name: {alias}
+     - Storage Account Folder: {alias}-files
+   - **Note:** check the standard quota for the file share, and adjust as needed. (CMI typically sets 5120 GB, or 5 TB)
```

Comment:
can we make this 1TB? I see no reason to expose ourselves to more risk without a reason.  We can always grow it for a community later, once we have a discussion with them.